woocommerce/wc-admin
# i18n: Advanced Filters strings

Add translation and context (including link to screenshot) to Advanced Filters such that translators can determine the appropriate translation and the filter is rendered in natural language order.

Relevant discussion in pxLjZ-4T6-p2 and #285 

## Screenshots

English
<img width="792" alt="screen shot 2018-09-25 at 4 29 02 pm" src="https://user-images.githubusercontent.com/1922453/45993243-0b540e00-c0e2-11e8-9674-0b634f3cf01c.png">

Japanese
<img width="794" alt="screen shot 2018-09-25 at 4 27 28 pm" src="https://user-images.githubusercontent.com/1922453/45993248-0ee79500-c0e2-11e8-8cc7-8f0cfd72bfcf.png">

## Test

1. Go to `/wp-admin/admin.php?page=wc-admin#/analytics/orders?filter=advanced` and poke around the advanced filters to discover any possible regressions.
2. Change the language to something other than English
3. Manually change the translation in the config line 47, https://github.com/woocommerce/wc-admin/blob/f86c16ca7d3af6c2b21c9cf8c3057e52a8fe25f9/client/analytics/report/orders/config.js#L47
to something else (apologies to any Francophiles):
```
{{rule /}} le Status de l'Ordre {{filter /}}
```
4. See the filter rendered accordingly
<img width="785" alt="screen shot 2018-09-25 at 4 21 01 pm" src="https://user-images.githubusercontent.com/1922453/45992640-12c5e800-c0df-11e8-80df-67ee5bbabf75.png">
5. Change the values of the filter and see the fieldset's legend update
<img width="275" alt="screen shot 2018-09-25 at 4 38 48 pm" src="https://user-images.githubusercontent.com/1922453/45993164-97b20100-c0e1-11e8-8be7-9b6ee232ce28.png">
6. Extra Credit: fire up VoiceOver and ensure all legends and labels are available


Fixes #285
